src/pages/networks/forms/NetworkForm.tsx

```diff
@@ -63,13 +63,14 @@ export interface NetworkFormValues {
   ipv6_ovn_ranges?: string;
   network?: string;
   yaml?: string;
+  type: "network";
```

Comment:
nit: the type attribute naming seems to be vague, maybe something like lxdEntityType might be better?

Reply:
entityType makes a lot of sense, I'll chagne it.